pkg/fileutil/fileutil.go

```diff
@@ -167,6 +168,11 @@ func CopyTree(src, dest string, uidRange *uid.UidRange) error {
 		return nil
 	}
 
+	fis, err := ioutil.ReadDir(dest)
```

Comment:
Do we really want this change? If we call CopyTree to copy some tree into a directory with some files there, then they will be deleted. Maybe we should rather have a separate function that deletes the old directory and then calls `CopyTree`?